Assets/BoostSlide.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BoostSlide : StateMachineBehaviour
+{
+    PlayerController pc;
+    public bool shorterDashCooldown;
```

Comment:
This is only going to be changed at editor-time, so it's safe to hard-set it to true or false (unless you're putting this on other state machine states).

Reply:
Just did that for visibility.